mixer/cmd/repos.go

```diff
@@ -66,12 +67,21 @@ var setURLRepoCmd = &cobra.Command{
 	Run:   runSetURLRepo,
 }
 
+var setExcludesRepoCmd = &cobra.Command{
+	Use:   "exclude <repo> <pkg> [<pkg>...]",
+	Short: "Exclude packages from a specified repo",
+	Long:  `Exclude packages from a specified repo. These packages will be ignored during build bundles. Globbing is supported.`,
```

Comment:
If you really are adding this functionality to mixer as opposed to letting users modify the yum conf themselves you should probably put the use case in the `Long` text here.

Reply:
Please still add the use-case here. Something like "Allows the user to explicitly select which packages to use when building bundles by excluding the unwanted ones"

Reply:
you didn't add the use case